src/lib/pkcs11/pkcs11t.h

```diff
@@ -973,6 +973,10 @@ typedef CK_ULONG          CK_MECHANISM_TYPE;
 #define CKM_RSA_PKCS_TPM_1_1           0x00004001UL
 #define CKM_RSA_PKCS_OAEP_TPM_1_1      0x00004002UL
 
+#define CKM_EDDSA_KEY_PAIR_GEN         0x00009040UL
+#define CKM_EDDSA                      0x00009041UL
+#define CKM_EDDSA_DERIVE               0x00009050UL
```

Comment:
Where are these mechanisms defined? Are they in any published PKCS#11 document?

Reply:
Nowhere/None. I sent a message to the OASIS PKCS#11 list asking if an action was planned but if I got a few "interested" answers nothing from a member of the PKCS#11 committee (so if you know one please push). BTW this falls into the IETF curdle WG scope too (i.e., it is a second way to get progress).

Reply:
Would you be interested in writing the proposal?

Reply:
yes but I have no good way to push it...

Reply:
Should we for now use something similar to this?

#define CKK_EDDSA		(CKK_VENDOR_DEFINED|0x00008003)